app/controllers/training_modules_controller.rb

```diff
@@ -20,5 +20,17 @@ def show
   def certificate
     @training_module = TrainingModule.find_by(name: params[:training_module_id])
     @module_progress = ModuleOverviewDecorator.new(helpers.module_progress(@training_module))
+
+    track('module_complete') if module_complete_untracked?
+
+    CalculateModuleState.new(user: current_user).call
```

Comment:
Was just thinking we can go a step further and bump this to `helpers.new_method_name`